examples/install/cluster-operator/01-ServiceAccount-strimzi-cluster-operator.yaml

```diff
@@ -1,12 +1,5 @@
-##---
-# Source: strimzi-kafka-operator/templates/01-ServiceAccount-strimzi-cluster-operator.yaml
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: strimzi-cluster-operator
-  labels:
-    app: strimzi-kafka-operator
-    chart: strimzi-kafka-operator-0.1.0
-    component: service-account
-    release: RELEASE-NAME
-    heritage: Tiller
+  labels: {}
```

Comment:
Why do we need all these `labels: {}`? Looks ugly and has no value. Can we just remove the whole `labels:` block?

Reply:
I agree it's ugly and has no value. We _could_ remove the whole `labels` object, but that would preclude us adding any labels of our own. We don't use any labels on these things currently (perhaps we should? I think something like `app: strimzi-kafka-operator` isn't unreasonable). Or I could continue removing the helm ones specificallty and then remove any `labels: {}` using `sed`. Thoughts?

Reply:
I think we are not using any labels now so having `labels: {}` there seems unneeded. I'm not necessarily against having some labels such as `app: stirmzi` or similar. So I think we should either add it or remove the empty labels.